tests/kernel_bundle/has_kernel_bundle_throws_diff_dev_from_one_platform_dev.cpp

```diff
@@ -36,8 +36,7 @@ class TEST_NAME : public sycl_cts::util::test_base {
     for (const auto &platform : platforms) {
       auto devices{platform.get_devices()};
       if (devices.size() <= 1) {
-        log.note("Platform skipped due to one device was found");
-        continue;
+        SKIP("Requires at least two devices");
       }
```

Comment:
Shouldn't we change the logic of this test?
Currently this test validates all platforms with 2+ devices. If I understand the sematic of SKIP correctly, new version will validate platforms with 2+ devices before the first one with one (or less?) device.

Reply:
Whoops - good catch, thanks. I made those changes somewhat mechanically and I did not see that loop there. Reverted.

---

As an aside, the logic here is not ideal. We shouldn't be performing tests on random devices that happen to be available. I think it's time we start to think about how conformance should work in regards to features that require multiple devices. I'll open an issue.